datafusion/sqllogictest/test_files/functions.slt

```diff
@@ -995,3 +995,8 @@ query ?
 SELECT find_in_set(NULL, NULL)
 ----
 NULL
+
+query B
```

Comment:
Providing context about what this test is doing might be helpful for the future. Something like

```suggestion
# Verify that multiple calls to volatile functions like `random()` are not combined / optimized away
query B
```